Woof/Woof/Helper/DataTransformer.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+/// A utility enum for transforming and formatting data.
+enum DataTransformer {
+    /// Generates a formatted full name by combining the given surname and name.
+    ///
+    /// - Parameters:
+    ///   - surname: The surname of the person.
+    ///   - name: The name of the person.
+    /// - Returns: The formatted full name in the format "name initial-of-surname."
+    /// If either the surname or the name is empty, an empty string is returned.
+    static func getFormattedNameFor(name: String, surname: String) -> String {
+        let fullName = "\(name) \(surname)".trimmingCharacters(in: .whitespacesAndNewlines)
+        return fullName
+    }
+
+    /// Converts a numeric value to an integer within the range of 0 to 5, following the zero-to-five style.
+    ///
+    /// - Parameter value: The numeric value to be converted.
+    /// - Returns: An integer value within the range of 0 to 5.
+    /// If the value is less than 0, 0 is returned.
+    /// If the value is greater than 5, 5 is returned.
+    /// Otherwise, the rounded value is returned. The rounding rule is schoolbook rounding.
+    static func convertValueZeroToFiveStyle(_ value: Double) -> Int {
```

Comment:
maybe it will improve readability?
```suggestion
    static func convertValueZeroToFiveRange(_ value: Double) -> Int {
```

Reply:
Agree, it seems more readable.

Reply:
Fixed with https://github.com/ios-course/ironfoudation-team-project/pull/38/commits/ab028a86d2106485e183354371d19c32d354165b, thanks for the ideas.